plugins/inventory_sync/module/gui/dialog_failed_download.lua

```diff
@@ -121,23 +124,24 @@ dialog_failed_download.events = {
 		end
 
 		if event.element.name == "inventory_sync_failed_download_abort" then
-			-- Give the player a charatcer and let them play with that
-			local player = game.get_player(event.player_index)
+			-- Give the player a character and let them play with that
+			local player = assert(game.get_player(event.player_index))
```

Comment:
Why the assert here?

Reply:
LuaLS complaining about nil check; I think we should take a decision on if we want to use `assert` or `--- @cast player -nil` or `--[[ @as LuaPlayer ]]`